projects/payment-lib/src/lib/components/case-transactions/case-transactions.component.ts

```diff
@@ -12,6 +12,7 @@ import {IRemission} from '../../interfaces/IRemission';
   styleUrls: ['./case-transactions.component.css']
 })
 export class CaseTransactionsComponent implements OnInit {
+  @Input() readOnly: boolean;
```

Comment:
Can you initialise readOnly from the PaymentLibComponent. Instead of duplicating the declaration.